src/website/src/app/documentation/demos/datagrid/datagrid.demo.module.ts

```diff
@@ -53,6 +54,13 @@ const routes: Routes = [
           demoName: 'Basic Structure',
         },
       },
+      {
+        path: 'usage',
```

Comment:
About extraction of the usage section - I like it, but I'd love to hear from other team members, too. And maybe I'd put it at the end of the list, not after Basic Structure.

Reply:
Agreed -- I think the end of the list (or possibly as the first item) would be better than the current location.

Reply:
Now showing this at the end.